.github/workflows/ci.yml

```diff
@@ -1,71 +1,48 @@
 name: Continuous Integration
 on: [push]
 
-jobs:
-  build:
-    name: Build and test
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - run: echo Hello World
 
-    
+jobs: 
+  Build:
+      name: Build and test
+      runs-on: ubuntu-latest
+      environment: TodoApp_Environment,Repository secrets
+      env:
+        WEBHOOK_URL: "${{ secrets.WEBHOOK_URL }}"
 
-    -  name: Login to DockerHub
-       uses: docker/login-action@v1 
-       with:
-          username: ${{secrets.DOCKERHUB_USERNAME}}
-          password: ${{secrets.DOCKERHUB_TOKEN}}
-      
-    -  name: Build and push
-       id: docker_build
-       uses: docker/build-push-action@v2
-       with:
-          push: true
-          tags: arpithadockerhub/app:latest
-
-    - name: Build Unit Tests
-      run: docker build --target test --tag todo-app:test .
-     
-
-    - name: Run Unit Test
-      run: docker run --env-file .env.test todo-app:test  /app/todo_app/tests
```

Comment:
We should still be running your unit tests (at least!) as part of this pipeline - switching them off for the sake of testing your deployment is reasonable but we should make sure we have them back before merging

Reply:
added mockmongo tests